boards/arm/cc3220sf_launchxl/cc3220sf_launchxl_defconfig

```diff
@@ -4,7 +4,6 @@ CONFIG_SOC_FAMILY_TISIMPLELINK=y
 CONFIG_SOC_SERIES_CC32XX=y
 CONFIG_SOC_CC3220SF=y
 CONFIG_CORTEX_M_SYSTICK=y
-CONFIG_FLASH=y
```

Comment:
I validated this for cc3220sf.  Looks good.

Reply:
Thanks for the confirmation.